core/deployment/src/main/java/io/quarkiverse/langchain4j/deployment/AiServicesProcessor.java

```diff
@@ -150,9 +148,9 @@ public class AiServicesProcessor {
 
     @BuildStep
     public void nativeSupport(CombinedIndexBuildItem indexBuildItem,
-            List<AiServicesMethodBuildItem> aiServicesMethodBuildItems,
-            BuildProducer<ReflectiveClassBuildItem> reflectiveClassProducer,
-            BuildProducer<ServiceProviderBuildItem> serviceProviderProducer) {
+                              List<AiServicesMethodBuildItem> aiServicesMethodBuildItems,
```

Comment:
Seems like there is a formatting change. Probably best to check your IDE, as I don't see a good reason for this to change